GoogleUtilities/Environment/Public/GoogleUtilities/GULNetworkInfo.h

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// The type of network that the device is running with. Values should correspond to the NetworkType
+/// values in android/play/playlog/proto/clientanalytics.proto
+typedef NS_ENUM(NSInteger, GULNetworkType) {
+  GULNetworkTypeNone = -1,
+  GULNetworkTypeMobile = 0,
```

Comment:
Should we call it GULNetworkTypeCellular? Mobile is too generic. We can start adding more types like HOTSPOT in future.

Reply:
I used Mobile to be consistent with the values these will map to: https://github.com/firebase/firebase-ios-sdk/blob/master/FirebasePerformance/ProtoSupport/Protos/perf_metric.proto#L390

Reply:
Sounds good!